peernode/src/peer-node/store/mod.rs

```diff
@@ -1,19 +1,31 @@
-use crate::peer::MarketClient;
 use crate::producer;
-use crate::transfer::files;
-use crate::transfer::jobs::Jobs;
-use anyhow::Result;
+use crate::transfer::files::LocalFileInfo;
+use crate::transfer::{
+    files::{self, get_file_info},
+    jobs::{self, Jobs},
```

Comment:
Nitpick: remove jobs::self unused import (unless it's used later then feel free to ignore this)